recipes/fastqc/meta.yaml

```diff
@@ -14,15 +14,15 @@ source:
 
 build:
   detect_binary_files_with_prefix: true
-  number: 3
+  number: 4
 
 requirements:
   run:
     # Enforce a version requirement on openjdk to ensure
     # it comes from the conda-forge channel and not default.
     # Many yaks were shaved to bring us this information.
     # Version number reference: https://github.com/conda/conda/issues/6948#issuecomment-369360906
-    - openjdk >8.0.121
+    - openjdk >=8.0.122
```

Comment:
That won't work. We just need https://github.com/bioconda/bioconda-recipes/issues/5026#issuecomment-401472985.

Reply:
Yeah, I realized that as well after this didn't work ;-) Thanks for following up!